src/components/Common/ListView/index.tsx

```diff
@@ -34,6 +38,18 @@ const ListView = ({
         </div>
       )}
       <ul className="cards-vertical">
+        {plexItems?.map((title, index) => {
+          return (
+            <li key={`${title.ratingKey}-${index}`}>
+              <TmdbTitleCard
+                id={title.tmdbId}
+                tmdbId={title.tmdbId}
+                type={title.mediaType}
+                canExpand
+              />
```

Comment:
If we ever have access to a TheTVDB ID, would be good to include it here as well to aid in identifying items that no longer exist on TMDB.

Reply:
We aren't returning tvdb id's with this data atm. Items that no longer exist in TMDB shouldn't come back from Plex's watchlist API. It would also be filtered out on our end if there was no TMDB ID.